devtools-ci-index.yaml

```diff
@@ -3251,6 +3251,20 @@
             git_repo: fabric8-analytics-license-analysis
         - '{ci_project}-{git_repo}-fabric8-analytics-pydoc':
             git_repo: fabric8-analytics-license-analysis
+        - '{ci_project}-{git_repo}-f8a-build-master':
+            git_organization: fabric8-analytics
+            git_repo: fabric8-analytics-notification-scheduler
+            ci_project: 'devtools'
+            ci_cmd: '/bin/bash cico_build_deploy.sh'
+            saas_git: saas-analytics
+            deployment_units: 'notification-scheduler'
+            deployment_configs: 'f8a-notification-scheduler'
+            timeout: '20m'
+            extra_target: rhel
```

Comment:
I don't see any Dockerfile.rhel file in the repository.

Reply:
this is supposed to be part of osio deployed on preview / prod? If so please make sure that rhel images will be used similar to other services